DedicatedServer/Scripts/Libs/Zrx/ModeLibs/FlagRush/UI/Modules/MVP.Script.txt

```diff
@@ -0,0 +1,327 @@
+// #RequireContext CMlScriptIngame
+#Include "Libs/Zrx/ModeLibs/FlagRush/UI/UIShared.Script.txt" as UIShared
+#Include "AnimLib" as AL
+
+Void Show() {
+	declare netwrite Integer Net_FlagRush_MVPStamp for Teams[0];
+	declare netwrite Boolean Net_FlagRush_MVPShow for Teams[0];
+	Net_FlagRush_MVPStamp = Now;
+	Net_FlagRush_MVPShow = True;
+}
+
+Void Hide() {
+	declare netwrite Integer Net_FlagRush_MVPStamp for Teams[0];
+	declare netwrite Boolean Net_FlagRush_MVPShow for Teams[0];
+	Net_FlagRush_MVPStamp = Now;
+	Net_FlagRush_MVPShow = False;
+}
```

Comment:
Only one of the lines is different in these functions. Could have just made one Function `Void Toggle(Boolean Visibility)`:

```suggestion
Void Toggle(Boolean Visibility) {
	declare netwrite Integer Net_FlagRush_MVPStamp for Teams[0];
	declare netwrite Boolean Net_FlagRush_MVPShow for Teams[0];
	Net_FlagRush_MVPStamp = Now;
	Net_FlagRush_MVPShow = Visibility;
}
```